app/views/welcome/index.html.erb

```diff
@@ -0,0 +1 @@
+<%= render 'next-event', next_event: @events.first%>
```

Comment:
In this commit (6f0ebb808cd58793e1e4d7d9824334a4cc4e3579) you've added a partial that doesn't exist yet.